app/webpack.common.js

```diff
@@ -17,8 +17,27 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.((png)|(eot)|(woff)|(woff2)|(ttf)|(svg)|(gif))(\?v=\d+\.\d+\.\d+)?$/,
-        loader: "file-loader?name=/[hash].[ext]"
+        test: /\.(jpg)|(png)|(svg)|(gif)$/,
```

Comment:
If you look at the rest of the file, I separated out the loader definitions so that fonts and images loaded by webpack are separated in the rendered site into "fonts" and "images" folders to match the hugo static configuration, so all these resources end up where they're supposed to be.